src/NuGet.Clients/NuGet.CommandLine/Common/CommandLineResponseFile.cs

```diff
@@ -64,7 +65,11 @@ private static string[] ParseArgsResponseFiles(string[] args, int parseArgsRespo
 
                 if (!File.Exists(responseFilePath))
                 {
-                    throw new ArgumentException(string.Format(LocalizedResourceManager.GetString("Error_ResponseFileDoesNotExist")), arg);
+                    //if the file could not be found, report this back to the user, but dont crash yet, the use of '@' in the argument might be intentional here.
+                    System.Console.WriteLine(string.Format(LocalizedResourceManager.GetString("Error_ResponseFileDoesNotExist"), arg));
```

Comment:
You may need to add additional validation to verify you are not printing the password back to the console